Makefile

```diff
@@ -58,7 +58,8 @@ FLAGS+=-DENABLE_DARWIN_EVENTS -DMAC_OSX_TK
 endif
 
 # TODO - remove dependency on GUI directory here
-FLAGS+=-std=c++11 -Ischema -Iengine -Imodel $(OPT) -UECOLAB_LIB -DECOLAB_LIB=\"library\" -Wno-unused-local-typedefs
+#FLAGS+=-std=c++11 -Ischema -Iengine -Imodel -IRESTService -I/home/wd/usr/json_spirit/include $(OPT) -UECOLAB_LIB -DECOLAB_LIB=\"library\" -Wno-unused-local-typedefs
+FLAGS+=-std=c++11 -Ischema -Iengine -Imodel -IRESTService -I/home/wd/usr/json_spirit/include $(OPT) -UECOLAB_LIB -DECOLAB_LIB=\"library\" -Wno-unused-local-typedefs
```

Comment:
If json_spirit were installed in a standard place (/usr, /usr/local or ~/usr), then this is unnecessary.

Reply:
I have reinstalled json in ~/usr, so no need to refer to the local include directory anymore.